windowsforms/matching-game/net8-windows/vb/MatchingGame/My Project/Resources.resx

```diff
@@ -46,7 +46,7 @@
     
     mimetype: application/x-microsoft.net.object.binary.base64
     value   : The object must be serialized with 
-            : System.Runtime.Serialization.Formatters.Binary.BinaryFormatter
+            : System.Serialization.Formatters.Binary.BinaryFormatter
```

Comment:
Previous namespace was correct: https://learn.microsoft.com/en-us/dotnet/api/system.runtime.serialization.formatters.binary.binaryformatter?view=net-8.0

Reply:
This file is automatically generated and because I used the VS Update tool to port the code from net45 to net8, it looks like it kept the "old" wording in that comment.